### PR TITLE
Add crawler check for invalid transition between two combats

### LINKED ISSUE #315: Crawler should use mid-combat roleplay flow
<a href="https://github.com/smartin015"></img></a> **Issue by [smartin015](https://github.com/smartin015)**
_Sunday Oct 08, 2017 at 12:55 GMT_
_Originally opened as https://github.com/ExpeditionRPG/expedition-quest-creator/issues/403 (2 comment(s))_

----

Follow starting node's parents up to combat node, as well as target node.
Compare both combat's data-line attribute to confirm equal / same combat.
Otherwise, error

### PR DESCRIPTION
**Pull request by smartin015**
Fixes #315

Note that this handles both combat cases mentioned in the bug, and the op evaluation errors were addressed in a previous github issue.

Also includes a test to catch accidentally committing `test.only` to the repo, as this was accidentally done in a couple places and we had tests we didn't know were failing.

### CODE REVIEW

File: services/quests/errors/definitions/427.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
+export const NUMBER = 427;
+export const NAME = `Detected an invalid transition from this combat to another combat`;
```
Comment (toddmedema) [inline services/quests/errors/definitions/427.tsx]: Can be streamlined to `Invalid transition from this...`

File: services/quests/errors/definitions/427.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+export const DESCRIPTION = `Jumping between combat nodes is not supported.`;
```
Comment (toddmedema) [inline services/quests/errors/definitions/427.tsx]: Nodes -> cards

File: services/quests/errors/definitions/427.tsx (hunk 1 above)
Comment (smartin015) [reply services/quests/errors/definitions/427.tsx]: Done.

File: services/quests/errors/definitions/427.tsx (hunk 2 above)
Comment (smartin015) [reply services/quests/errors/definitions/427.tsx]: Done.

**Pull request merged**